import_data/invoke.yaml

```diff
@@ -26,6 +26,21 @@ osm:
   ## Path of the file
   file:
 
+wikidata:
+  sitelinks:
+    url: https://github.com/QwantResearch/wikimedia-dumps/releases/download/10/2019/sitelinks.csv.gz
```

Comment:
You download all wikimedia stuff from october 2019? :o

Reply:
I'll give more explicit names to these dumps, this one actually includes data from November to mid October.